content/en/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -101,7 +101,10 @@ afterwards to tell Kubernetes that it can resume scheduling new pods onto the no
 ## Draining multiple nodes in parallel
 
 The `kubectl drain` command should only be issued to a single node at a
-time. However, you can run multiple `kubectl drain` commands for
+time, because as the node is unscheduled the newly created pods are scheduled on different
```

Comment:
I don't understand what this mean.

What unschedules a node? I haven't heard of unscheduling.

Reply:
@sftim ok, trying to find better word for proper understanding